### ISSUE TITLE
Gynogenesis

### ISSUE THREAD

**Issue by ASEMBER2** [Tree-of-Sex/toso#79]
### In which category is your new term request?

- [X] new_definition
- [ ] existing_definition

### Preferred term label

gynogenesis

### Synonyms

sperm-dependent parthenogenesis or pseudogamy

### Synonym type

None

### Textual definition

A clonal (asexual) mode of reproduction. In gynogenetic systems female produces unreduced ova (by ameiotic process or endoreduplication) that develop into all-female offspring. As opposed to parthenogenesis, gynogenesis is dependent of sperm of parental sexual lineage or related sexual species to trigger embryonic development. Sperm donors usually do not contribute genetically to the next generation as the sperm genome gets eliminated after fertilization. Thus, in contrary to parthenogenesis, gynogenetic organisms have to co-exist with a sexual species, which serve as a sperm donator. These associations are reffered to as complexes. Occasionally, the sperm nuclear and/or mitochondrial genome may be incorporated and contribute genetically to the progeny (paternal leakage), in the range from subgenomic amounts to entire sperm genome incorporation. The latter case results in an elevated ploidy level in the offspring (genome addition). Definition summarized from: Lamatsch DK, Stöck M. 2009 (https://doi.org/10.1007/978-90-481-2770-2) and Stöck M, et al. 2021 (https://doi.org/10.1098/rstb.2020.0103).

### Parent term

asexual (BCIO:015004; NCIT:C155696), asexuality (GSSO:000699), Reproduction, Asexual (OMIT:0013024). Other parent term should be: clonal reproduction (not yet defined in OLS)

### Children term(s)

gynogen, paternal leakage

### Taxon-restricted

Seems to be restricted to teleost fishes thus far. Gynogenesis occurs e.g. in Cobitis loaches, Amazon molly Poecilia formosa, Prussian carp Carassius gibelio, Carassius lansdorfii, Phoxinus eos, Poeciliopsis. 

### Reference

Lamatsch DK, Stöck M. 2009 Sperm-dependent parthenogenesis and hybridogenesis in teleost fishes. In Lost sex—The evolutionary biology of parthenogenesis (eds I Schön, K Martens, P. van Dijk), pp. 399–432. Heidelberg, Berlin, Germany:
Springer. https://doi.org/10.1007/978-90-481-2770-2

Stöck M, Dedukh D, Reifová R, Lamatsch DK, Starostová Z, Janko K. 2021 Sex chromosomes in meiotic, hemiclonal, clonal and polyploid hybrid vertebrates: along the ‘extended speciation continuum’. Phil. Trans. R. Soc. B 376: 20200103.
https://doi.org/10.1098/rstb.2020.0103

### Additional context

Related mechanism: kleptogenesis.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by RoelMWouters** [comment]
In the textual definition I disagree with the first sentence: A clonal (asexual) mode of reproduction. As the brackets synonimize clonal with asexual. I would change to "A form of asexual reproduction with clonal progeny"

**Comment by kareljanko1** [comment]
**Comment on “gynogenesis” term**

Keep the core of the definition _**strictly matroclinous and mechanism-agnostic**_. “Gynogenesis” should be _**defined by female descent (matrocliny)**_ plus sperm dependence for egg activation, not by how unreduced eggs are produced (there are multiple pathways). Classic usage explicitly ties gynogenesis to matroclinal, clonal inheritance with sperm needed only to trigger development, while noting that a variety of cytological routes can produce unreduced ova; therefore the definition should not mention specific mechanisms like endoreduplication/ameiosis. 

**Suggested textual definition (concise):**
Gynogenesis is a matroclinous, clonal reproductive mode in which females produce eggs that require sperm to activate embryogenesis, but the sperm genome is typically excluded so daughters are genetically identical to the mother. Rare “paternal leakage” can occur, from subgenomic fragments to whole-genome incorporation, which may yield elevated ploidy (“genome addition”). 
 

**Synonyms (with types):**

Exact: sperm-dependent parthenogenesis (for female-only systems). 

Related: pseudogamy (zoological sense; avoid as exact because the term is used differently in botany). 

Parent terms: asexual reproduction; clonal reproduction (process).

Related/child concepts: paternal leakage (related process); genome addition (outcome of rare paternal incorporation). 

**Scope/taxon note:** Do not restrict to teleosts. Sperm-dependent parthenogenesis (including gynogenesis sensu stricto for female systems) has evolved repeatedly across animals (multiple phyla; numerous genera). Use “gynogenesis” for female-limited systems; for hermaphroditic sperm-dependent systems prefer the broader process label “sperm-dependent parthenogenesis.” Examples of well-documented gynogens include the Amazon molly Poecilia formosa and several fish/loach complexes (e.g., Cobitis). 
 

**Rationale & key refs to cite alongside current ones:**
Beukeboom & Vrijenhoek (1998) https://doi.org/10.1046/j.1420-9101.1998.11060755.x for the matrocliny-based definition, synonymy, and mechanism-agnostic stance; Janko et al. (2023)[(https://doi.org/10.1002/ece3.10522)for modern synthesis and clear treatment of paternal leakage/genome addition with figures and case studies (e.g., Poecilia, Cobitis).

**Comment by lukkrat** [comment]
I would suggest a more concise definition:
Gynogenesis is a matroclinous reproductive mode in which females produce eggs that require sperm to activate embryogenesis, but the sperm genome is typically excluded.

**Comment by Lucija27** [comment]
> **Comment on “gynogenesis” term**

A very small rewording suggestion: Gynogenesis is a matroclinous, clonal reproductive mode in which females produce eggs that require sperm to activate embryogenesis, but the sperm genome is typically excluded, making daughters genetically identical to the mother. 

I would put the second part - Rare “paternal leakage” can occur, from subgenomic fragments to whole-genome incorporation, which may yield elevated ploidy (“genome addition”). - as the comment below it, instead of in the definition itself.

**Comment by lukkrat** [comment]
> > **Comment on “gynogenesis” term**
> 
> A very small rewording suggestion: Gynogenesis is a matroclinous, clonal reproductive mode in which females produce eggs that require sperm to activate embryogenesis, but the sperm genome is typically excluded, making daughters genetically identical to the mother.
> 
> I would put the second part - Rare “paternal leakage” can occur, from subgenomic fragments to whole-genome incorporation, which may yield elevated ploidy (“genome addition”). - as the comment below it, instead of in the definition itself.

I agree with putting the details into comment. But I am not sure whether "clonal" is essential in the definition, dropping it allows space for at least theoretically possible asexual nonclonal production of eggs.

**Comment by ASEMBER2** [comment]
I agree with:
Gynogenesis is a matroclinous, clonal reproductive mode in which females produce eggs that require sperm to activate embryogenesis, but the sperm genome is typically excluded, making daughters genetically identical to the mother.

I would put (with small modification "mayL" instead of "can") the following to "Additional context" section:
Rare “paternal leakage” may occur, from subgenomic fragments to whole-genome incorporation, which may yield elevated ploidy (“genome addition”).
